wiki/Article_styling_criteria/Formatting/en.md

```diff
@@ -621,7 +621,6 @@ There are two types of image links: inline and reference. Examples:
 ```markdown
 ![][flag_AU]
 
-[flag_AU]: /wiki/shared/flag/AU.gif
```

Comment:
there is no redirect, is there?

Reply:
Hmm, I have the impression that I made a small mistake in the end, I had to remove the two redirects instead of one, because the flags update had added a link at the bottom of the page, and it was him who created the error, not that one. This will be corrected on the next commits.